packages/document-model-libs/.storybook/main.ts

```diff
@@ -1,10 +1,8 @@
-import { getConfig } from "@powerhousedao/codegen";
 import type { StorybookConfig } from "@storybook/react-vite";
 
-const { editorsDir } = getConfig();
```

Comment:
This doesn't work?

Reply:
yeah at least on my machine, says there's a problem with cjs vs mjs 😢